tutorials/setup.md

```diff
@@ -1,3 +1,25 @@
+![CRI-O logo](../logo/crio-logo.svg)
+- [Build and install CRI-O from source](#build-and-install-cri-o-from-source)
+  * [Runtime dependencies](#runtime-dependencies)
+  * [Build and Run Dependencies](#build-and-run-dependencies)
+    + [Fedora 31 RHEL 7 CentOS](#fedora-31-rhel-7-centos)
+    + [RHEL8](#rhel8)
+    + [Debian Raspbian Ubuntu](#debian-raspbian-ubuntu)
+  * [Get Source Code](#get-source-code)
+  * [Build](#build)
+    + [Build Tags](#build-tags)
+  * [Static builds](#static-builds)
+    + [Creating a release archive](#creating-a-release-archive)
+  * [Download conmon](#download-conmon)
+  * [Setup CNI networking](#setup-cni-networking)
+  * [CRI-O configuration](#cri-o-configuration)
+    + [Validate registries in registries.conf](#validate-registries-in-registriesconf)
+    + [Recommended - Use systemd cgroups.](#recommended---use-systemd-cgroups)
+    + [Optional - Modify verbosity of logs](#optional---modify-verbosity-of-logs)
+    + [Optional - Modify capabilities and sysctls](#optional---modify-capabilities-and-sysctls)
+  * [Starting CRI-O](#starting-cri-o)
+  * [Using CRI-O](#using-cri-o)
+
 # Build and install CRI-O from source
```

Comment:
I think it'd look better if the title was above the TOC. Either above or below the image is fine with me

Reply:
if you agree and go through with that, you could also drop the L2 (` - [Build and install CRI-O from source](#build-and-install-cri-o-from-source) `), and then move every indent in one